esp8266-driver.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/esp8266-driver/#264468722a97cc2c8e0a517da06a542eb52fb9cb
+https://github.com/ARMmbed/esp8266-driver/#c2b7e15c08ca37aa08323e0a9198fca63cda50e2
```

Comment:
Should not touch the ESP8266 driver.

Reply:
This occurred when I did the “git pull origin master —rebase” as requested. Can you not do the merge or does this merge from origin make it incorrect? Is it normal to have a mergerequest volleyed back and forth so many tomes or is there perhaps a unspoken message here?

Reply:
@TeemuKultala & @jannehekkala, please clearly communicate what it is you would like me to do.

Reply:
50e2 is the same revision as master.  What's the recommended way to resolve this?  I think a merge at this point would end up in the same state as master.

git log --oneline -- esp8266-driver.lib
e2de261 Update ESP8266 driver with bugfixes and new features
cc599e5 Update ESP8266 driver with bugfixes and performance improvements
540d43e ESP8266 update
70acdad ESP8266 driver update - drop fix
1626163 ESP8266 - allow overriding timeouts
57711c2 ESP8266 - gethostbyname fix
0620b8d ESP8266 v1.3 lib
a7ca6d1 ESP8266 driver updates
d350340 (origin/revert-mts) Revert "Add mts driver"
e8b993a Add mts driver
e61ba69 (origin/mbedos5.3.4) Bump atmel-rf-driver and esp8266-driver
15a293e Update to esp8266 master, atmel-rf master and add mcr20a driver
5a733bd Pin esp8266 driver because mbed OS 5.1.2 cannot compile against it
dd245e4 Change WIFI to ESP8266_WIFI
fbcc0ac Easy Connect initial commit

 git show e2de261
commit e2de261bb85df4b32df62fae7ea74a2213c3a0d9
Author: Veijo Pesonen <veijo.pesonen@arm.com>
Date:   Wed Feb 28 09:59:08 2018 +0200

    Update ESP8266 driver with bugfixes and new features

diff --git a/esp8266-driver.lib b/esp8266-driver.lib
index 2fbc62c..aaf273c 100644
--- a/esp8266-driver.lib
+++ b/esp8266-driver.lib
@@ -1 +1 @@
-https://github.com/ARMmbed/esp8266-driver/#264468722a97cc2c8e0a517da06a542eb52fb9cb
+https://github.com/ARMmbed/esp8266-driver/#c2b7e15c08ca37aa08323e0a9198fca63cda50e2

Reply:
@bridadan @janjongboom any suggestions?

Reply:
Looks like the history is pretty goofed up. I think the easiest way would be to cherry-pick your good commit on top of the latest master. To do that, run the following commands:


```
# Add the upstream if you haven't already done so
git remote add upstream https://github.com/ARMmbed/easy-connect.git

# Fetch upstream master
git fetch upstream master

# Create a new branch to add your changes
git checkout -b add_wnc14a2a upstream/master

# Cherry-pick the good commit on top of the new branch
git cherry-pick c980290f5f6b9ed10df828dbd43db8c5a862ebf9
```

At this point you should have a good branch named `add_wnc14a2a` that you can push to your fork. When you do just create a new PR and we can close this one.